### DISCUSSION TITLE
No `Location` field when creating/updating transactions

### DISCUSSION THREAD

**Discussion by bedlamzd** [firefly-iii/firefly-iii#6805] [Firefly III] [answered]
Can't figure out if this feature supposed to work or not.

So there is an option in preferences to enable `Location` field 

<details>

<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/44726005/210911516-80f6afcc-8b8f-4f6c-9650-3cfa62f307f7.png)

</details>

and there are map-related variables in `.env` file of which I set `ENABLE_EXTERNAL_MAP` to true.

After restarting my docker container I see no such option when creating transactions, although env var updated.

I've searched through docs, issues and discussions and couldn't find anything useful besides #4400.

Am I missing something? Am I supposed to somehow configure external map service? Or is it still in progress?

**Comment by bedlamzd** [comment]
Since then I've found that you need a mapbox api key (which I added to my env) and found that tags can have a location (and that works).

But for transactions it's not. And option I've mentioned appears to do nothing.

@JC5 can you please clarify if this should work or it's just a stub for the future?

**Reply by JC5** [reply, marked as answer]
It's currently not enabled, it's a stub as you say. Sorry for the late reply!

**Reply by bedlamzd** [reply]
Got it, thank you!
